package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
- [ ] I recommend adding Webpack `start` and `watch` scripts to make `npm start` possible to start the app. You can add the following lines of code in your scripts : `"watch": "webpack --watch",`
    `"start": "webpack serve --open",`